ee-security/pom.xml

```diff
@@ -250,29 +239,19 @@
                     <plugin>
                         <groupId>org.wildfly.plugins</groupId>
                         <artifactId>wildfly-maven-plugin</artifactId>
-                        <version>${version.wildfly.maven.plugin}</version>
                         <configuration>
                             <feature-packs>
-                                <feature-pack>
+                                <feature-pack>                                  
                                     <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
                                 </feature-pack>
-                                <feature-pack>
-                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.cloud.fp}</location>
+                                <feature-pack>                                    
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.pack.cloud}</location>
                                 </feature-pack>
                             </feature-packs>
                             <layers>
                                 <layer>cloud-server</layer>
                             </layers>
                             <filename>ROOT.war</filename>
-                            <packaging-scripts>
```

Comment:
Think you still need this for this QS, it reconfigures elytron for the provisioned server

Reply:
Fixed!